asf_heat_pump_affordability/pipeline/archetypes.py

```diff
@@ -0,0 +1,102 @@
+import pandas as pd
+from asf_heat_pump_affordability import config
+
+
+def classify_dict_archetypes_masks(
```

Comment:
Here I tried to give the function some flexibility - i.e. ages_mapping can be updated if desired - but hard coded the property types rather than overcomplicate the code. If archetypes change, the code will need to be updated or a new function added.

The alternative option I could think of was hard coding helper functions for each archetype with year split and then having one `classify_dict_archetypes_masks()` function which generates the dict using only helper functions and no hard coding.

Ultimately I went with the choice here in the hopes that it is slightly more flexible and would reduce amount of code changes if archetypes change slightly.

Reply:
I'm happy with this as is. for me the important thing is that you could in theory supply any mapping of archetypes-boolean filter and generate a result.